src/App.js

```diff
@@ -1,64 +1,42 @@
-import React, { Component } from 'react';
-import PropTypes from 'prop-types';
-import './App.css';
-import HeaderForm from './components/HeaderForm';
-import TubeMapContainer from './components/TubeMapContainer';
-import CustomizationAccordion from './components/CustomizationAccordion';
-import { dataOriginTypes } from './enums';
-import * as tubeMap from './util/tubemap';
-import config from './config.json';
+import React, { Component } from "react";
```

Comment:
It looks like you've reformatted everything automatically to use double quotes, whereas previously the convention in the codebase was mostly single quotes.

Are double quotes really better? Are they the default of the popular formatter you're using and so we should switch because that's what popular formatters do?

Reply:
They are the default of Prettier, the formatter I'm using, which just automatically added double quotes. I don't have any strong opinion on this, and after Googling it seems there's not a consistent rule. If you'd like I can revert to single quotes?